easybuild/easyconfigs/n/netcdf4-python/netcdf-python-1.4.1-CrayGNU-18.08-python2.eb

```diff
@@ -0,0 +1,33 @@
+#
+name = 'netcdf-python'
+version = '1.4.1'
+
+homepage = 'http://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+
+source_urls = ['https://pypi.python.org/packages/source/n/netCDF4/']
+sources = ['netCDF4-%(version)s.tar.gz']
+
+req_py_majver = 2
+req_py_minver = 7
+
+pythonver = '%s.%s' % (req_py_majver, req_py_minver)
+versionsuffix = '-python%s' % req_py_majver
+
+dependencies = [
+    ('cray-python/2.7.15.1', EXTERNAL_MODULE),
+    ('netCDF-Fortran', '4.4.4'),
```

Comment:
I think normally you should be able to use the cray-netcdf instead (includes netcdf-fortran):
('cray-netcdf/4.6.1.2', EXTERNAL_MODULE),

Reply:
I took it from the 1.2.9 recipe.

Reply:
In either CDO or NCO (don't remember), I changed it to use `cray-netcdf` instead of  netCDF-Fortran and the performance increased significantly. So I think, if it works, you should use `cray-netcdf` (also it simplifies the recipe).

Reply:
I'll try that.